docs/3.0.0-beta.x/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,25 @@
+# 💬 Troubleshooting
+
+Below are solutions to some common issues that you may experience when working with Strapi. You can also post questions to [StackOverflow](https://stackoverflow.com/questions/tagged/strapi) or reach out to the members of our [Slack](https://slack.strapi.io) community!
+
+## Frequently Asked Questions
+
```

Comment:
I would add more questions to the FAQ - maybe from https://strapi.io/blog/top-10-questions-to-strapi-ceo

Reply:
Move FAQ stuff to my FAQ PR: https://github.com/strapi/strapi/pull/5376